CHANGELOG.md

```diff
@@ -4,7 +4,15 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [0.23.1] - unreleased
+## [0.23.2]
+
+### Added
+
+- `Family::get_or_create_clone` can access a metric in a labeled family. This
+   method avoids the risk of runtime deadlocks at the expense of incrementing
+   a reference count.
```

Comment:
```suggestion
   method avoids the risk of runtime deadlocks at the expense of creating an owned type.
```

Given that `M` is generic, calling `to_owned` on it might use some other synchronization mechanism than reference counting.

Reply:
this wording is addressed in 4c24551.